src/import/FSHTank.ts

```diff
@@ -402,6 +402,10 @@ export class FSHTank implements Fishable {
             // don't break; keep looping in case there is a later rule that re-assigns url
           }
         });
+        // If there is no URL-setting rule, return the URL that will be auto-generated on export
+        if (!meta.url && result.usage !== 'Inline') {
+          meta.url = `${this.config.canonical}/${result.instanceOf}/${meta.id}`;
```

Comment:
I'm glad you thought of this! Commit [e999747](https://github.com/FHIR/sushi/pull/1100/commits/e9997478f0b5c35319621ee2b9a189dd28a81583) moves this functionality from the Tank to the Fisher so that we can generate the url correctly.